<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>585</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Video Annotation Tool</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonFollowStyle</enum>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="MouseLabel" name="image">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>741</width>
      <height>501</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="menuWidget" native="true">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>10</y>
      <width>181</width>
      <height>181</height>
     </rect>
    </property>
    <widget class="QPushButton" name="save">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="openVideo">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Open Video</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="buttonBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>181</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>Menu</string>
     </property>
    </widget>
    <widget class="QPushButton" name="openImages">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Open Images</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loadXML">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Load XML</string>
     </property>
    </widget>
    <zorder>buttonBox</zorder>
    <zorder>openImages</zorder>
    <zorder>openVideo</zorder>
    <zorder>save</zorder>
    <zorder>loadXML</zorder>
   </widget>
   <widget class="QWidget" name="buttonWidget" native="true">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>520</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <widget class="QPushButton" name="next">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>0</y>
       <width>91</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Next &gt;&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="prev">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>91</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;&lt; Prev</string>
     </property>
    </widget>
    <widget class="QPushButton" name="associate">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>0</y>
       <width>91</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Associate</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hide">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>0</y>
       <width>91</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Hide</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>0</y>
       <width>91</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="frameLabel">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>520</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="selectId">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>200</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Select Id</string>
    </property>
    <widget class="QComboBox" name="idBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>181</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>944</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MouseLabel</class>
   <extends>QLabel</extends>
   <header location="global">mouselabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
